### ISSUE TITLE
Re-implemented SSD has high loss and low acc during training

### ISSUE THREAD

**Issue by Yurodidon** [rykov8/ssd_keras#158]
I implemented the SSD follow this project, but when I was training on PASCAL VOC2007 dataset, the acc became smaller and smaller, at the same time the loss is around 4.0.

The codes are available on https://github.com/CeLaMbDa/keras_SSD

**Comment by Ryo-Kawanami** [comment]
I'm also same situation...
I have trained during 16 epoch, but loss, acc, val_loss, val_acc are almost same.

**Comment by Ryo-Kawanami** [comment]
Could anybody who knows this situation help?

**Comment by Yurodidon** [comment]
Hi Ryo-Kawanam, I think I found the reason of this situation!
Check these:
1. Do not use metrics in training, I don't know why, but still, when I delete those metrics, the network works much better.
2. You should compile model after freezing layers, this was pointed out in the official document, please refer to https://keras.io/getting-started/faq/#how-can-i-freeze-keras-layers
3. Use a fit learning rate, the can't be to small or too big, if it is too small it might lead the weights can't change.
